libraries/AP_InertialNav/AP_InertialNav_NavEKF.cpp

```diff
@@ -30,6 +31,14 @@ void AP_InertialNav_NavEKF::update()
     // get the velocity relative to the local earth frame
     Vector3f velNED;
     if (_ahrs_ekf.get_velocity_NED(velNED)) {
+        // during high vibration events use vertical position change
+        // ToDo: fix and use EKF's get_vert_pos_rate() function instead of baro
```

Comment:
todo fixed?

Reply:
yes, txs.  comment removed in follow-up commit.